src/components/Strategies/StrategyList.tsx

```diff
@@ -1,59 +1,39 @@
 import { Card, Row, Col, Typography } from 'antd';
 import styled from 'styled-components';
 
+import { IStrategies } from './../../types';
+
+import { strategies } from './StrategyData';
+
 const { Meta } = Card;
 
 const { Text } = Typography;
 
 function StrategyList() {
+  const strategyClick = (strategy: IStrategies) => {
+    var comingSoonTextId = document.getElementById(strategy.title);
+    if (strategy.displayComingSoonText) {
+      comingSoonTextId!.style.display = 'block';
+    } else {
+      window.open('/' + strategy.detailPageURL);
+    }
+  };
+
   return (
     <Container>
       <Row gutter={[24, { xs: 8, sm: 16, md: 24, lg: 32 }]}>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/atlas-mine.jpg" />}>
-            <Meta title="Claim Rewards" description="Bridgeworld (Atlas Mine)" />
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/magic-dragon.jpg" />}>
-            <Meta title="Claim Rewards" description="Magic Dragon DAO" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/battlefly.jpg" />}>
-            <Meta title="Claim Rewards" description="BattleFly" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/tales-of-alleria.jpg" />}>
-            <Meta title="Claim Rewards" description="Tales of Elleria" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/smithy.jpg" />}>
-            <Meta title="Go on Quests" description="SmithyDAO" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card className="antcard" hoverable cover={<img alt="example" src="../img/ivory-tower.jpg" />}>
-            <Meta title="Send Legions Questing" description="Bridgeworld" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
+        {strategies.map((strategy) => (
+          <Col span={8} key={strategy.itemId}>
+            <div onClick={() => strategyClick(strategy)}>
```

Comment:
Looks like you had to define custom events in Heap for each type of click. Would probably be easier if you just called e.g. `heap.track('StrategyClick', { name, url })`